pkg/engine/harpoon.go

```diff
@@ -758,6 +764,14 @@ func (hc *HarpoonConfig) getPathOrTree(target *api.Target, subDir, method string
 	return "", subDirTree, err
 }
 
+func checkInitialErr(name, method string, err error) {
```

Comment:
why distinguish between initial repo processing vs non-initial processing? Is it worth separate funcs that are almost identical?